lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -29,7 +29,7 @@ def source_dir
 
 def fixtures(category)
   begin
-    fixtures = YAML.load_file(".fixtures.yml")["fixtures"]
+    fixtures = YAML.load_file( ENV['FIXTURES_YML'] || '.fixtures.yml' )['fixtures']
```

Comment:
I just wanted to hop in and indicate that this would be VERY helpful on consulting engagements. And, as Chris mentioned, all of the other variables are not namespaced so there's really no precedence.

What would it take to get this pushed through?